src/hooks/useDate/useDate.ts

```diff
@@ -0,0 +1,20 @@
+import dayjs from 'dayjs'
+import { useEffect } from 'react'
+import { selectSelectedLocale } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+export function useDate() {
+  const selectedLocale = useAppSelector(selectSelectedLocale)
+
+  useEffect(() => {
+    require(`dayjs/locale/${selectedLocale}.js`)
+    const localizedFormat = require('dayjs/plugin/localizedFormat')
+    dayjs.extend(localizedFormat)
```

Comment:
This could move to a global dayjs configuration place instead of the hook

Reply:
I'm not conceptually against using a hook. although it would create a new instance for every component consumption and would theoretically configure dayjs multiple times instead of one, that's a cheap enough operation.

Reply:
What about configuring dayjs localizedFormat either in the RTK preferences slice, or the `onClick` that dispatches it?

https://github.com/shapeshift/web/blob/396a05bcc8931446ed8352d5b6138f450e7551b2/src/state/slices/preferencesSlice/preferencesSlice.ts#L43
https://github.com/shapeshift/web/blob/47da0e056eefce23c7d7f85b242c946d902d89e7/src/components/Modals/Settings/Languages.tsx#L60

I think the latter would be cleaner, as we don't really have a side effects model: slices are effectively (pure or not) methods which ultimate goal is to return data/update a slice